buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -480,6 +479,7 @@ class BeamModulePlugin implements Plugin<Project> {
         google_cloud_datacatalog_v1beta1            : "com.google.cloud:google-cloud-datacatalog:$google_cloud_datacatalog_version",
         google_cloud_dataflow_java_proto_library_all: "com.google.cloud.dataflow:google-cloud-dataflow-java-proto-library-all:0.5.160304",
         google_cloud_datastore_v1_proto_client      : "com.google.cloud.datastore:datastore-v1-proto-client:1.6.3",
+        google_cloud_pubsublite                     : "com.google.cloud:google-cloud-pubsublite:$google_cloud_pubsublite_version",
```

Comment:
Google cloud pubsublite depends on grpc 1.29 and has other deps that differ greatly from the common set used within Beam.

Please validate that no additional linkage errors were produced using the [linkage checker](https://github.com/apache/beam/blob/master/sdks/java/build-tools/beam-linkage-check.sh) for all impacted modules and post the results here. You will need to resolve linkage errors by either downgrading the version of the library used to one that is compatible with the current versions in Beam or upgrading Beam's dependencies till they are compatible with the library.

Reply:
Will run this, this tool requires a git repo, while I'm using hg-git for these. IDK if its possible to make this not dependent on the source control tool? In the meantime, I'll clone my repo using git and run the tool.

Reply:
This tool fails to run in a git clone of the branch. Please fix this tool?

```
/bin/bash ./sdks/java/build-tools/beam-linkage-check.sh
Updating files: 100% (4678/4678), done.
HEAD is now at 9304d1c90 Check for null attribute map in IncomingMessage before adding on PubsubMessage.
Mon Jun 22 10:43:24 EDT 2020: Installing artifacts of master(9304d1c9) to Maven local repository.
beam ❯ 
```

Reply:
@suztomo for awareness.

Do you have anything special within your `.m2` settings?

Reply:
I guess the error went to /dev/null. Checking dpcollins-google:pubsublite_beam branch.

Reply:
I had to run "gradlew clean". Then it went well; no new linkage errors in dpcollins-google:pubsublite_beam branch. Good. https://gist.github.com/suztomo/57e888b95556e5741cced3999313a2f8 

I'll update the script so that it should not swallow error messages https://issues.apache.org/jira/browse/BEAM-10298

As per dependency on Git, because the tool compares two Git branches for their dependencies, it's coupled with Git.

Reply:
I think this is resolved as it doesn't cause dependency conflicts.